official/vision/beta/projects/basnet/train.py

```diff
@@ -0,0 +1,72 @@
+# Lint as: python3
```

Comment:
can you follow this example to write the train.py? https://github.com/tensorflow/models/blob/master/official/vision/beta/projects/vit/train.py